plugins/org.eclipse.n4js/src/org/eclipse/n4js/external/NpmPackageToProjectAdapter.java

```diff
@@ -117,12 +118,19 @@ public boolean apply(Path path) {
 				// create marker file to denote that this package was among "namesOfPackagesToAdapt"
 				// (compare with: ExternalProjectLocationsProvider#isExternalProjectDirectory(File))
 				File markerFile = new File(packageRoot, N4JSGlobals.PACKAGE_MARKER);
-				Files.write(markerFile.toPath(), Collections.singletonList( // will overwrite existing file
-						"Temporary marker file. See N4JSGlobals#PACKAGE_MARKER for details."));
+				String content = "Temporary marker file. See N4JSGlobals#PACKAGE_MARKER for details.";
+				Files.write(markerFile.toPath(), Collections.singletonList(content)); // will overwrite existing file
+
+				File newPackageRoot = new File(typeDefFolder, packageRoot.getName() + "-n4jsd");
+				if (!newPackageRoot.exists()) {
+					newPackageRoot.mkdir();
+				}
+				File markerFile2 = new File(newPackageRoot, N4JSGlobals.PACKAGE_MARKER);
+				Files.write(markerFile2.toPath(), Collections.singletonList(content)); // will overwrite existing file
```

Comment:
as discussed: I would've expected to not create this marker in case no type definitions exists for the current project, but it's only an intermediate state anyway